### DISCUSSION TITLE
Building from Source

### DISCUSSION THREAD

**Discussion by shawnwildermuth** [ebkr/r2modmanPlus#1392] [General]
I have a tiny PR I want to send, but I can't get the project to build. Can you confirm what versions of Node you're using (and Python if that's necessary). I'm on a Windows machine but it didn't work on WSL (e.g. Linux either). I can share the crash info, but I thought it might be more helpful if I knew what environment you were using with Quasar.

**Comment by yobson1** [comment]
A bit late but in case anyone else comes across this
I had some problems initially too. It says to use node 13 in the building.md but there's a dependency that requires >=14 so use that instead. Using node 14 just run `yarn install` then `yarn run run` or `yarn build-win`, I didn't install quasar globally either the npm scripts suffice.
